ja_jp/README.adoc

```diff
@@ -0,0 +1,252 @@
+[float]
```

Comment:
The contents is the same as README.adoc in root. Can we remove `ja_jp/README.adoc`?
If we put readme in ja_jp, it should contain Japanese-userguide-specific description.

Reply:
Removed.